README.rst

```diff
@@ -371,7 +372,7 @@ libsonata is distributed under the terms of the GNU Lesser General Public Licens
 unless noted otherwise, for example, for external dependencies.
 Refer to `COPYING.LESSER` and `COPYING` files for details.
 
-Copyright (C) 2018-2020, Blue Brain Project/EPFL and contributors.
+Copyright (c) 2018-2020 Blue Brain Project/EPFL
```

Comment:
Changing copyright would require `+2` from all contributors! 😅

Reply:
@alex4200 what is the guidance on this?

Reply:
from what I can see, all the past committers are part of BBP. The IP does not belong to them but BBP/EPFL.
So you can remove that "and contributors."